### PR TITLE
SQLStudio: Execute scripts and show results

### PR DESCRIPTION
**Pull request by martinfalisse**
Adds button to execute scripts and a table to display the results.
Screenshot from 2022-06-26 17-16-40

### CODE REVIEW

File: Userland/DevTools/SQLStudio/MainWidget.cpp [hunk 1]
```diff
@@ -135,6 +137,10 @@ MainWidget::MainWidget()
         update_editor_actions(editor);
     });
 
+    m_run_script_action = GUI::Action::create("Run script", { Mod_Alt, Key_F9 }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-forward.png").release_value_but_fixme_should_propagate_errors(), [&](auto&) {
```
Comment (linusg) [inline Userland/DevTools/SQLStudio/MainWidget.cpp:143]: I think the play icon would be more appropriate, Hack Studio uses a similar icon for its run action

File: Userland/DevTools/SQLStudio/MainWidget.cpp [hunk 2]
```diff
@@ -138,7 +139,16 @@ MainWidget::MainWidget()
     });
 
     m_run_script_action = GUI::Action::create("Run script", { Mod_Alt, Key_F9 }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-forward.png").release_value_but_fixme_should_propagate_errors(), [&](auto&) {
-        TODO();
+        auto editor = dynamic_cast<ScriptEditor*>(m_tab_widget->active_widget());
+        if (!editor)
+            return;
+        // TODO select the database to use in UI.
+        auto connection_id = m_sql_client->connect("test");
+        auto sql_statements = editor->document().text().split(';');
```
Comment (linusg) [inline Userland/DevTools/SQLStudio/MainWidget.cpp:150]: What if there is a semicolon inside a string, or a comment? This won't work. can the SQL client not deal with multiple statements at once?

**Review by linusg — CHANGES_REQUESTED** [review]
Nice :)
Typo in first commit: `so that can`

File: Userland/DevTools/SQLStudio/MainWidget.cpp (hunk 2 above)
Comment (martinfalisse) [reply Userland/DevTools/SQLStudio/MainWidget.cpp]: True.. I changed the implementation, now I've kind of copied what was in the sql.cpp utility. Still thinking perhaps I should factor out the code.. into the SQLClient?

File: Userland/DevTools/SQLStudio/MainWidget.cpp [hunk 3]
```diff
@@ -135,6 +142,14 @@ MainWidget::MainWidget()
         update_editor_actions(editor);
     });
 
+    m_run_script_action = GUI::Action::create("Run script", { Mod_Alt, Key_F9 }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/program-run.png").release_value_but_fixme_should_propagate_errors(), [&](auto&) {
```
Comment (linusg) [inline Userland/DevTools/SQLStudio/MainWidget.cpp:145]: I didn't mean using the Hack Studio icon for this, which only makes sense next to the others (see screenshot) - just use the [play icon](https://github.com/SerenityOS/serenity/blob/master/Base/res/icons/16x16/play.png) please, as suggested.

image

**Pull request merged**